superset/db_engine_specs/presto.py

```diff
@@ -729,6 +732,11 @@ def get_create_view(
     def handle_cursor(cls, cursor: Any, query: Query, session: Session) -> None:
         """Updates progress information"""
         query_id = query.id
+        poll_interval = (
+            query.database.get_extra()
```

Comment:
it would be nice to add a property `connect_args` to the db object as this code becomes a bit repetitive:
https://github.com/apache/incubator-superset/search?q=connect_args&unscoped_q=connect_args

and by the way connect args should be part of the engine_params